web/themes/custom/votegov/templates/layout/page.html.twig

```diff
@@ -115,12 +115,13 @@
 
   {% include '@votegov/component/contact-identifier.html.twig' %}
 
+  {% include '@votegov/component/footer--secondary-section.html.twig' %}
+
   <div class="usa-footer__primary-section" data-test="footer">
     <div class="grid-container">
       {# Footer menu. #}
       {{ drupal_entity('block', 'votegov_footernavigation') }}
     </div>
   </div>
```

Comment:
Let's move the markup into the menu template. so in this file it would just look like:
```
{# Footer menu. #}
{{ drupal_entity('block', 'votegov_footernavigation') }}
```